CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.2.0] - 2024-10-16
+### Changed
+- Updated all dependency versions for hive, hadoop, datanucleus, reflections, guava, tez, mockito, junit and added them in pom.xml <properties> section
+- Updated all maven plugin versions
```

Comment:
You can leave this without specifying the version or just remove it completely us the end users of HiveRunner shouldn't need to care about what Maven plugins were used to build it.